From 4e0fda512da4178049cb660742eab707848f32d0 Mon Sep 17 00:00:00 2001
From: "Darryl L. Pierce" <mcpierce@apache.org>
Date: Fri, 24 May 2013 14:58:37 +0000
Subject: [PATCH 5/6] QPID-4885: C++ examples install to qpid/examples

Previously they installed to /usr/local/share/examples. This changes
that to /usr/local/share/qpid/examples instead.

git-svn-id: https://svn.apache.org/repos/asf/qpid/trunk@1486084 13f79535-47bb-0310-9956-ffa450edef68
---
 qpid/cpp/BuildInstallSettings.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/qpid/cpp/BuildInstallSettings.cmake b/qpid/cpp/BuildInstallSettings.cmake
index d63a61e..8146c3b 100644
--- a/qpid/cpp/BuildInstallSettings.cmake
+++ b/qpid/cpp/BuildInstallSettings.cmake
@@ -93,7 +93,7 @@ if (WIN32)
     "Directory to install SASL configuration files")
   set (QPID_INSTALL_DATADIR conf CACHE STRING
     "Directory to install read-only arch.-independent data root")
-  set (QPID_INSTALL_EXAMPLESDIR examples CACHE STRING
+  set (QPID_INSTALL_EXAMPLESDIR qpid/examples CACHE STRING
     "Directory to install programming examples in")
   set (QPID_INSTALL_DOCDIR docs CACHE STRING
     "Directory to install documentation")
@@ -174,7 +174,7 @@ if (UNIX)
   set (QPID_INSTALL_CONFDIR ${SYSCONF_INSTALL_DIR})
   set (QPID_INSTALL_SASLDIR ${SYSCONF_INSTALL_DIR}/sasl2)
   set (QPID_INSTALL_DATADIR ${SHARE_INSTALL_DIR}/qpid)
-  set (QPID_INSTALL_EXAMPLESDIR ${SHARE_INSTALL_DIR}/examples)
+  set (QPID_INSTALL_EXAMPLESDIR ${SHARE_INSTALL_DIR}/qpid/examples)
   set (QPID_INSTALL_DOCDIR ${DOC_INSTALL_DIR}) # Directory to install documentation
   set (QPID_INSTALL_INCLUDEDIR ${INCLUDE_INSTALL_DIR})
   set (QPID_INSTALL_LIBDIR ${LIB_INSTALL_DIR})
-- 
1.8.1.4

